packages/ava-react/src/NarrativeTextVis/utils/getPrefixCls.ts

```diff
@@ -1 +1,3 @@
-export const getPrefixCls = (suffixCls?: string) => `ntv-${suffixCls}`;
+export function getPrefixCls(suffixCls?: string) {
+  return `ntv-${suffixCls}`;
```

Comment:
The cls prefix here should have `ava-react-` or `ar-` but `ntv-` is used before. Let's just keep that 🤔

Reply:
Will it be changed later?

Reply:
I think we need change it. We can determine a specification that holds the `ntv-` prefix for now.

Reply:
TODO